repositories.bzl

```diff
@@ -24,7 +24,7 @@ def req_compile_dependencies():
 
     maybe(
         py_requirements_repository,
-        name = "req_compile_sdist_compiler",
+        name = "sdist_deps",
```

Comment:
Should keep the name so related repositories are similarly named.

Reply:
Ugh what a workspace thing to say